html/index.html

```diff
@@ -6,7 +6,55 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
-<body>
-    
-</body>
+  <body>
+    <header>
+      <h1>Marte Auer</h1>
+      <nav id="nav">
+        <a href="#about-me">About Me</a>
+        <a href="#contact">Contact</a>
+        <a href="#skills">Skillset</a>
+      </nav>
+    </header>
+
+    <main>
+      <section id="about-me">
+        <h2>About me</h2>
+        <p>
+          Hi there! Welcome to my personal website. Here is gathered the most
+          important information about me. If you want to work with me or get to
+          know me, please don't hesitate to get in contact.
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eligendi quibusdam ipsa alias fugiat sequi, necessitatibus recusandae quaerat delectus corrupti illo. Eaque, libero in quis similique aspernatur nihil nulla iure quo recusandae voluptatum perspiciatis earum rerum maxime, nam cumque sunt ullam quae vitae! Perspiciatis illum aut repudiandae possimus rem provident doloremque facere laboriosam dignissimos delectus, exercitationem repellendus harum eligendi quaerat, dicta nemo molestias! Dignissimos enim cupiditate eaque sequi esse! Doloremque ipsum a necessitatibus praesentium harum et ab iure voluptatibus quo, hic optio quos nesciunt quisquam, reiciendis perferendis sapiente error voluptate! Vel molestiae iste perferendis voluptates iure pariatur corporis repellat deserunt blanditiis?
+        </p>
+
+        <figure>
+          <img
+            src="../assets/IMG_20210224_183654.jpg"
+            alt="picture of a whippet"
```

Comment:
```suggestion
            alt="a whippet"
```
you want to avoid adding "image of/picture of" when adding a description to an image. The screenreader would read "an image of picture of whippet", so it is doppelt gemoppelt :P